### PR TITLE
Migrate the library to use the structure proposed by cli

### PR DESCRIPTION
**Pull request by alfonsserra**

### CODE REVIEW

File: projects/showcase/browserslist [hunk 1]
```diff
@@ -9,4 +9,4 @@
 last 2 versions
 Firefox ESR
 not dead
-IE 9-11 # For IE 9-11 support, remove 'not'.
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```
Comment (apenalver2) [inline projects/showcase/browserslist]: Does that mean we are not going to support IE 11? I know that this is for the showcase, but in general, the components are going to support IE11?

Comment (alfonsserra) [reply projects/showcase/browserslist]: Reverted!!!!! Thanks Aritz.

**Comment by alfonsserra** [comment]
@vinyulis Regarding to the polyfills, I have seen that are needed for NgClass support on SVG elements and Web Animations `@angular/platform-browser/animations`.
Am I right?

**Comment by vinyulis** [comment]
> @vinyulis Regarding to the polyfills, I have seen that are needed for NgClass support on SVG elements and Web Animations `@angular/platform-browser/animations`.
> Am I right?

Yes it seems, so then we don't need to uncomment nothing in polyfills file.

**Pull request merged**